records_mover/records/records_format.py

```diff
@@ -60,7 +60,7 @@ def __init__(self,
         """
         self.format_type = 'delimited'
         self.variant = variant
-        self._custom_hints = hints
+        self.custom_hints = hints
```

Comment:
This isn't documented (and thus public for outside code to use), but it is now used by some of the target code.